src/org/zaproxy/zap/extension/requester/RequesterNumberedTabbedPane.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ * 
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License. 
+ * You may obtain a copy of the License at 
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0 
+ *   
+ * Unless required by applicable law or agreed to in writing, software 
+ * distributed under the License is distributed on an "AS IS" BASIS, 
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+ * See the License for the specific language governing permissions and 
+ * limitations under the License. 
+ */
+package org.zaproxy.zap.extension.requester;
+
+import org.zaproxy.zap.extension.requester.ManualHttpRequestEditorPanel;
+
+public class RequesterNumberedTabbedPane extends NumberedTabbedPane {
+
+	private static final long serialVersionUID = 1L;
+
+	public RequesterNumberedTabbedPane() {
+		super();		
+	}
+
+	@Override
+	public void addDefaultTab() {
+		ManualHttpRequestEditorPanel requestPane = new ManualHttpRequestEditorPanel(true, "resend", "ui.dialogs.manreq");
```

Comment:
I think it should use it's own configuration key, not the one from the "Resend" dialogue.
From the point of view of a user I wouldn't expect the changes done in the dialogue to affect the Requester tab.